### PR TITLE
USERPROV-1189 Adding documentation for V4.1 GET User and updating V4 with latest new fields

### PR DESCRIPTION
**Pull request by edwinanto2003**
Adding documentation for V4.1 GET User and updating V4 with latest new fields

### CODE REVIEW

**Comment by MarcBell-SAP** [comment]
@edwinanto2003 The user edwin-anto you used to create the first two commits is not associated with your GitHub account. Please use the following instructions to correct this:

You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=962) it.

**Comment by MarcBell-SAP** [comment]
Did you try using the instructions provided to fix the problem?

You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=962) it.

File: src/api-reference/user-provisioning/spend/v4.spend-user-get-started.md
Comment (wujasmine) [inline src/api-reference/user-provisioning/spend/v4.spend-user-get-started.md:21]: Missing a period in front of v4 line 21? 🙂 (.v4.spend-user-extension-schema.html#invoice-preference-extension-schema)

**Review by wujasmine — APPROVED** [review]

**Comment by RoAnnC1** [comment]
Hi @edwinanto2003, I am checking with @mvahlsing98117 on the status of the v4.1 version and when it's supposed to go live. Because there are edits to a published version in this PR those will automatically go live the next time we publish the site. So this is temporarily on hold until I hear from Matt.

File: src/api-reference/user-provisioning/spend/v4.1.spend-user-retrieval.md
Comment (eddyclhung) [inline src/api-reference/user-provisioning/spend/v4.1.spend-user-retrieval.md:61]: The Hostame here should be either US2 or EU2.

File: src/api-reference/user-provisioning/spend/v4.1.spend-user-retrieval.md
Comment (eddyclhung) [inline src/api-reference/user-provisioning/spend/v4.1.spend-user-retrieval.md:90]: Same for these location URLs in the response.

**Comment by RoAnnC1** [comment]
Hi @edwinanto2003 @mvahlsing98117 Thanks for resolving the comments. I'll defer to @MarcBell-SAP on the json part of this project, but will merge this next week and go through it.

**Comment by mvahlsing98117** [comment]
With the updates, it looks better. Ok to release

**Comment by MarcBell-SAP** [comment]
:shipit:

**Review by MarcBell-SAP — APPROVED** [review]
:shipit:

**Pull request merged**
